.github/workflows/performance.yml

```diff
@@ -23,8 +23,14 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  LOCAL_DIR: build
+  # This workflow takes two sets of measurements — one for the current commit,
+  # and another against a consistent version that is used as a baseline measurement.
+  # This is done to isolate variance in measurements caused by the GitHub runners
+  # from differences caused by code changes between commits. The BASE_TAG value here
+  # represents the current version being used for baseline measurements.
```

Comment:
See my comment in https://github.com/WordPress/wordpress-develop/pull/4139#discussion_r1123703622, I think this comment is great, but I think we should further clarify that this should not be updated to always be the current WordPress version.

I think the comment as is looks _almost_ good in that sense, just that last bit on "represents the current version being used" may be misunderstood as if it should be the current WordPress version. Maybe we can clarify that further?

Reply:
@felixarntz Do you think this is more clear?

```
  # This workflow takes two sets of measurements — one for the current commit,
  # and another against a consistent version that is used as a baseline measurement.
  # This is done to isolate variance in measurements caused by the GitHub runners
  # from differences caused by code changes between commits. The BASE_TAG value here
  # represents the version being used for baseline measurements. It should only be
  # changed if we want to normalize results against a different baseline.
```

Reply:
@joemcgill Thanks, that sounds a lot better to me 👍

Reply:
Updated in ccd09a0